### PR TITLE
Add libdef for metalsmith 2.x.x

### PR DESCRIPTION
**Pull request by jirutka**
https://github.com/segmentio/metalsmith

### CODE REVIEW

File: definitions/npm/metalsmith_v2.x.x/flow_v0.75.x/metalsmith_v2.x.x.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+// flow-typed signature: aa45d9abfcd7b277c466aed186e7a6d8
+// flow-typed version: <<STUB>>/metalsmith_v^2.3.0/flow_v0.75.0
```
Comment (GAntoine) [inline definitions/npm/metalsmith_v2.x.x/flow_v0.75.x/metalsmith_v2.x.x.js]: Remove these two lines.

File: definitions/npm/metalsmith_v2.x.x/test_metalsmith.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+// @flow
```
Comment (GAntoine) [inline definitions/npm/metalsmith_v2.x.x/test_metalsmith.js]: Move this test file into the `flow_v0.75.x-` folder.

File: definitions/npm/metalsmith_v2.x.x/test_metalsmith.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import Metalsmith from 'metalsmith';
+import type { DoneCallback, File, FilesMap } from 'metalsmith';
+
+import * as fs from 'fs';
+
```
Comment (GAntoine) [inline definitions/npm/metalsmith_v2.x.x/test_metalsmith.js]: Remove extra empty line.

**Comment by jirutka** [comment]
I did the requested changes and rebased.

**Pull request merged**